### PR TITLE
[Nice to have in 2.4] Fix/2038  fix description change not refresh

### PR DESCRIPTION
**Pull request by metzgegu**
#2038

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

**Comment by metzgegu** [comment]
Cypress tests don't fails locally

File: functionnal_tests/cypress/integration/app_workspace_advanced/operation_workspace_advanced.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        cy.resetDB()
+        cy.setupBaseDB()
+    })
+
+    beforeEach(function () {
+        cy.loginAs('administrators')
+        cy.visitPage({pageName: p.DASHBOARD, params: { workspaceId }})
+    })
+
+    it('Change workspace description', function () {
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/integration/app_workspace_advanced/operation_workspace_advanced.js]: the it() should be more detailed about the test it checks.
It should be something like:
``` javascript
describe('App Workspace Advanced', () => {
  describe("Changing the workspace's description"), () => {
    it('Should update the description in the dashboard', () => {
      // the test
    })
  })
})
```

**Comment by Skylsmoi** [comment]
Reminder: This PR will require a merge with a squash

File: functionnal_tests/cypress/integration/app_workspace_advanced/operation_workspace_advanced.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        cy.loginAs('administrators')
+        cy.visitPage({pageName: p.DASHBOARD, params: { workspaceId }})
+    })
+    describe("Changing the workspace's description", () => {
+        it('Should update the description in the dashboard', function () {
+            cy.getTag({selectorName: s.WORKSPACE_DASHBOARD})
+                .find('.dashboard__header__advancedmode__button.btn')
+                .click()
+
+            cy.wait(1000)
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/integration/app_workspace_advanced/operation_workspace_advanced.js]: this cy.wait should be avoidable

**Pull request merged**